app/views/repositories/show.html.erb

```diff
@@ -8,7 +8,11 @@
 
 <section id="show-repository">
 
+
   <section id="repository-photos">
+    <% if @repository.youtube_link %>
+      <%= youtube_video(@repository.youtube_link) %>
```

Comment:
Could you please also add the youtube_link field to the Edit for repositories? So they can edit it if they need to.

Reply:
Done!

Reply:
Great!